UDPClientGUI/ViewModels/StringToColorConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace UDPCommGUI.ViewModels
+{
+    class StringToColorConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return value.ToString() == "Connected" ? new SolidColorBrush(Colors.Green) : new SolidColorBrush(Colors.Red);
```

Comment:
Last time I asked you to implement this with an IValueConverter.

Now try doing the same with DataTrigger.

Reply:
Implemented